### PR TITLE
Update Module 10 assets section

### PR DESCRIPTION
**Pull request by millidavids**

### CODE REVIEW

**Comment by millidavids** [comment]
@bkuhl @jlaswell @Dylan-Buth

**Comment by Dylan-Buth** [comment]
:+1: Looks great. Thanks for changing that.

File: Module-10.md [hunk 1]
```diff
@@ -38,17 +38,8 @@ _Now would be a good time to review the difference between `{{` and `{!!` in Bla
...
-  ````
-  <html>
-    <head>
-      <title>Test</title>
-      {!! \RealpageLouisville\GuiLibrary\Asset::container('mypackage')->styles(); !!}
-      {!! \RealpageLouisville\GuiLibrary\Asset::container('mypackage')->scripts(); !!}
-    </head>
-  ````
-3. **README**: Your package must include a `README.md` file written in Markdown syntax that includes documentation on how to install your package as well as code usage examples.
+1. **Assets**: Your package should contain at least one (1) CSS stylesheet and one (1) Javascript file. The implementation of the Javascript and CSS is up to you. These assets should be available to the rest of your application. See [this reference](https://laravel.com/docs/5.1/packages#public-assets).
```
Comment (bkuhl) [inline Module-10.md]: Can you update this to reference `master` instead of `5.1`?

**Comment by bkuhl** [comment]
:+1: Thanks for taking care of that.  This is a great move

**Pull request merged**